Dockerfile

```diff
@@ -271,7 +318,8 @@ RUN jira completion bash > /etc/bash_completion.d/jira
 RUN oc completion bash > /etc/bash_completion.d/oc
 RUN ocm completion > /etc/bash_completion.d/ocm
 RUN osdctl completion bash > /etc/bash_completion.d/osdctl
-RUN rosa completion bash > /etc/bash_completion.d/rosa
+# rosa is only available for amd64 platforms so ignore it
+RUN [[ $(platform_convert "@@PLATFORM@@" --amd64 --arm64) != "amd64" ]] && rm ${BIN_DIR}/rosa || rosa completion bash > /etc/bash_completion.d/rosa
```

Comment:
Reviewers take note - there will be NO `rosa` binary inside the arm64 version, that's the only binary that doesn't have a pre-built version.

Reply:
Personally, I think I would prefer a dummy script that prints out "`rosa` CLI not supported on arm64. Please do X instead." and then exits with a non-zero status code. The argument against that idea is any script that runs `command -v rosa` will succeed when its use is guaranteed to fail.